cpp/src/Ice/SSL/SSLEndpointI.cpp

```diff
@@ -12,10 +12,32 @@ using namespace std;
 using namespace Ice;
 using namespace Ice::SSL;
 
+namespace
+{
+    const char* const sslPluginName = "IceSSL";
+}
+
 extern "C"
```

Comment:
I wonder if there is any point in keeping `extern "C"`for SSL, which is always loaded from Ice library. The only use case I see is if you were loading IceSSL with a different name (not clear this worked) you will now get this error. Otherwise the plug-in is already registered and no need for extern C.

Reply:
Fixed as suggested: the built-in plug-in functions for tcp, ssl, udp, ws and iap are no longer extern "C".